src/me/StevenLawson/TotalFreedomMod/Commands/Command_trail.java

```diff
@@ -18,16 +18,42 @@
 import org.bukkit.plugin.RegisteredListener;
 
 @CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off]")
+@CommandParameters(description = "Pretty rainbow trails.", usage = "/<command> [off | glass]")
```

Comment:
@Wild1145 Maybe, but depending on the parameters used, this command might result in either a glass trail or a wool trail. Therefore, implying it would be a glass trail makes no sense. :P